docs/index.md

```diff
@@ -18,11 +17,451 @@ focusing on the Generalized Linear Model (GLM).
 
 We provide a **Poisson GLM** for analyzing spike counts, and a **Gamma GLM** for calcium or voltage imaging traces.
 
-The package is under active development and more methods will be added in the future.
+<div class="grid cards">
```

Comment:
These links are all incorrect (except the LICENSE one). I think they're wrong because mkdocs normally updates them from this stub to the correct version but, because they're in html, mkdocs isn't touching them.